docs/docs/testing-react-components.md

```diff
@@ -2,41 +2,41 @@
+> Quanto mais seus testes se assemelharem à maneira como seu software é usado, mais confiança eles podem oferecer.
 
-It provides light utility functions on top of `react-dom` and `react-dom/test-utils` and gives you the confidence that refactors of your component in regards to the implementation (but not functionality) don't break your tests.
+Ele fornece funções leves de utilidade em cima do `react-dom` e `react-dom/test-utils` e lhe dá a confiança de que os refatores do seu componente em relação à implementação (mas não à funcionalidade) não quebrem seus testes.
 
-## Installation
+## Instalação
 
-Install the library as one of your project's `devDependencies`. Optionally you may install `jest-dom` to use its [custom jest matchers](https://github.com/testing-library/jest-dom#custom-matchers).
+Instale a biblioteca como uma das `devDependencies` do seu projeto. Opcionalmente, você pode instalar o `jest-dom` para usar seus [matchers personalizados do jest](https://github.com/testing-library/jest-dom#custom-matchers)..
 
 ```shell
 npm install --save-dev @testing-library/react @testing-library/jest-dom
 ```
 
-Create the file `setup-test-env.js` at the root of your project. Insert this code into it:
+Crie o arquivo `setup-test-env.js` na raiz do seu projeto. Insira este código nele:
 
 ```js:title=setup-test-env.js
 import "@testing-library/jest-dom/extend-expect"
 ```
 
-This file gets run automatically by Jest before every test and therefore you don't need to add the imports to every single test file.
+Esse arquivo é executado automaticamente pelo Jest antes de cada teste e, portanto, você não precisa adicionar as importações a cada arquivo de teste.
 
-Lastly you need to tell Jest where to find this file. Open your `jest.config.js` and add this entry to the bottom after 'setupFiles':
+Por fim, você precisa informar ao Jest onde encontrar esse arquivo. Abra seu `jest.config.js` e adicione esta entrada na parte inferior após 'setupFiles':
 
 ```js:title=jest.config.js
 module.exports = {
   setupFilesAfterEnv: ["<rootDir>/setup-test-env.js"],
 }
 ```
 
-## Usage
+## Uso
 
-Let's create a little example test using the newly added library. If you haven't done already read the [unit testing guide](/docs/unit-testing) — essentially you'll use `@testing-library/react` instead of `react-test-renderer` now. There are a lot of options when it comes to selectors, this example chooses `getByTestId` here. It also utilizes `toHaveTextContent` from `jest-dom`:
+Vamos criar um pequeno exemplo de teste usando a biblioteca recém-adicionada. Se você ainda não leu o [guia de teste de unidade][unit testing guide](/docs/unit-testing) — essencialmente você usará `@testing-library/react` em vez de `react-test-renderer` agora. Existem muitas opções quando se trata de seletores, este exemplo escolhe `getByTestId` aqui. Ele também utiliza `toHaveTextContent` do `jest-dom`:
 
 ```js
```

Comment:
Eu acho que você poderia traduzir o código abaixo na parte de texto. Por exemplo: Gatsby is awesome! => Gatsby é incrível!

Reply:
Pode fazer tradução do código? Neste caso está em duas partes do código:

`const Title = () => <h1 data-testid="hero-title">Gatsby is awesome!</h1>`
e
`expect(getByTestId("hero-title")).toHaveTextContent("Gatsby is awesome!")`

Reply:
Dúvida: Poderia traduzir os comentários dentro do código também?

`// You have to write data-testid`
`// Assertion`
`// --> Test will pass`

Reply:
Na minha opinião, acho que fica legal traduzir essas coisinhas, já que elas acabam ajudando as pessoas, sim!

Reply:
Ok, vou fazer essas alterações 👍